apps/client/src/components/compounds/GroupView/ExploreGroupView/index.tsx

```diff
@@ -0,0 +1,104 @@
+import { Button, Text } from 'components'
+import styles from './exploreGroup.module.scss'
+import frame from './images/Frame.svg'
+import rightless from './images/rightless.svg'
+import TableComponent from '../GroupTable'
+import CardComponent from '../GroupCard'
+
+const textArray = [
+  'Bros/Sis, no dull yourself for dis service year money mata Join Lagos Corp Members Thrift Saving Group sharp-sharp!We go secure your money, make am grow well-well, and support each other for dis money journey.Learn money sense, plan for your future, and enjoy the benefits of our community. No time to waste, come join us now! Contact [07067482633].Make we flex our money together!',
+]
```

Comment:
This is not necessary, you can just pass it to the text component

think like you are consuming an api